buildSrc/src/main/java/Config.kt

```diff
@@ -36,6 +36,7 @@ object Config {
         val espressoCore = "androidx.test.espresso:espresso-core:3.2.0"
         val androidxOrchestrator = "androidx.test:orchestrator:$androidxTestVersion"
         val mockitoKotlin = "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+        val apacheCommonsIo = "commons-io:commons-io:2.6"
```

Comment:
it's a `testImplementation` lib, I know, but our test classes are written in Kotlin and could be easily replaced by, eg:

```
val input = "{}"
val inputStream = ByteArrayInputStream(input.toByteArray(Charsets.UTF_8))
```

maybe even creating a kotlin extension method like `String.toInputStream()` which does the code above internally.

Reply:
Sure that works for me. Thanks!